docs/narr/sessions.rst

```diff
@@ -91,26 +91,32 @@ Remember that sessions should be short-lived and thus the number of clients affe
 .. code-block:: python
     :linenos:
 
+    import pickle
     from pyramid.session import JSONSerializer
-    from pyramid.session import PickleSerializer
     from pyramid.session import SignedCookieSessionFactory
 
+
     class JSONSerializerWithPickleFallback(object):
         def __init__(self):
             self.json = JSONSerializer()
-            self.pickle = PickleSerializer()
 
-        def dumps(self, value):
+        def dumps(self, appstruct):
+            """Accept a Python object and return bytes."""
             # maybe catch serialization errors here and keep using pickle
             # while finding spots in your app that are not storing
             # JSON-serializable objects, falling back to pickle
-            return self.json.dumps(value)
+            return self.json.dumps(appstruct)
 
-        def loads(self, value):
+        def loads(self, bstruct):
+            """Accept bytes and return a Python object."""
             try:
-                return self.json.loads(value)
+                return self.json.loads(bstruct)
             except ValueError:
-                return self.pickle.loads(value)
+                try:
+                    return pickle.loads(bstruct)
+                # at least ValueError, AttributeError, ImportError but more to be safe
```

Comment:
This comment seems misplaced. Maybe indent to align with the `return` or push below within the next code block?

Reply:
thanks for catching these so fast. I was having some issues with rst documents.  I'll address asap

Reply:
@stevepiercy so, I looked at this and I agree it is weird, but I copied this style exactly from the deprecated code:  https://github.com/Pylons/pyramid/blob/master/src/pyramid/session.py#L60-L66

I would be happy to nest the comment under the `except` block, as I prefer that style myself. this is up to you and Mike though, and I am happy to issue a pull on it if you'd like to switch it.

Reply:
Well if both of us had to say "huh?", then it is probably not the way to do it. :wink: